src/components/Dashboard/Carousel/Slide/Slide.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Slide = ({ children }) => {
+  return (
+    <div className="dp-carousel-slide active" data-order="first">
```

Comment:
With the active class hardcoded all slides will be active by default right?

Reply:
If you see a preveview [here](https://cdn.fromdoppler.com/doppler-webapp/demo-build3906/#/dashboard) all slides are in fact visible. And the carousel is not working. 👀

Reply:
And they are both orange 
![image](https://user-images.githubusercontent.com/2439363/134419349-2326c8ef-2463-47b5-aee6-dfae945de850.png)

Reply:
No, we will make it dynamic with Junior in another pr

Reply:
Then perhaps why not use the active class in one slide only?

Reply:
Yes, now the slides are generated dynamically, for that reason the active class is in all

Reply:
Nevermind is part of the functionality of active and my comment won`t work.